tools/External Tools.xml

```diff
@@ -112,4 +112,11 @@
       <option name="REGEXP" value=".*$FILE_PATH$:$LINE$.*" />
     </filter>
   </tool>
+  <tool name="VectorCAST coverage Current" description="Create VectorCAST coverage report on currently loaded package" showInMainMenu="false" showInEditor="false" showInProject="false" showInSearchPopup="false" disabled="false" useConsole="true" showConsoleOnStdOut="false" showConsoleOnStdErr="false" synchronizeAfterRun="true">
+    <exec>
+      <option name="COMMAND" value="_apex_clion_vc_coverage" />
+      <option name="PARAMETERS" value="$ProjectName$" />
+      <option name="WORKING_DIRECTORY" value="$ProjectFileDir$/.." />
```

Comment:
@dongilchang Where  value="$ProjectFileDir$/.."  pointing to? \
It seems not where CLION keeps his build folder "$CMakeCurrentBuildDir$"

Reply:
@MichaelOrlov I replied to your question in this link https://gitlab.apex.ai/ApexAI/grand_central/-/issues/9449#note_525463. Thank you.